copycat.tmux

```diff
@@ -6,6 +6,22 @@ source "$CURRENT_DIR/scripts/variables.sh"
 source "$CURRENT_DIR/scripts/helpers.sh"
 source "$CURRENT_DIR/scripts/stored_search_helpers.sh"
 
+# check if tmux supports keytable (v2.1+)
+check_keytable_support() {
+	if (( $(echo "$(tmux -V | cut -f2 -d' ') >= 2.1" | bc) )); then
+		keytable_supported=true
+	fi
+}
+
+# this function defines the default key binding table
+set_keytable() {
+	local keytable="$(get_tmux_option "$copycat_keytable_option" "$default_keytable")"
+	local search_value="$(tmux show-options -gqv "$copycat_keytable_option")"
+	if [ -z $search_value ]; then
+		tmux set-option -g "$copycat_keytable_option" "$keytable"
+	fi
+}
+
```

Comment:
Hey, why is this function needed? Can't we just use

    key_table="$(get_tmux_option "$copycat_keytable_option" "$default_keytable")"

wherever we need it?